tiflash/tiflash-configuration.md

```diff
@@ -89,16 +140,111 @@ minmax_index_cache_size = 5368709120 # 数据块 min-max 索引的内存 cache 
 [server]
     engine-addr = 外部访问 tiflash coprocessor 服务的地址
 [raftstore]
-    snap-handle-pool-size = 控制处理 snapshot 的线程数，默认为 2。设为 0 则关闭多线程优化
-    store-batch-retry-recv-timeout = 控制 raft store 持久化 WAL 的最小间隔。通过适当增大延迟以减少 IOPS 占用，默认为 4ms，设为 0ms 则关闭该优化。
+    ## 控制处理 snapshot 的线程数，默认为 2。设为 0 则关闭多线程优化
+    snap-handle-pool-size = 2
+    ## 控制 raft store 持久化 WAL 的最小间隔。通过适当增大延迟以减少 IOPS 占用，默认为 4ms，设为 0ms 则关闭该优化。
+    store-batch-retry-recv-timeout = "4ms"
 ```
 
 除以上几项外，其余功能参数和 TiKV 的配置相同。需要注意的是：`tiflash.toml [flash.proxy]` 中的配置项会覆盖 `tiflash-learner.toml` 中的重合参数；`key` 为 `engine` 的 `label` 是保留项，不可手动配置。
 
 ### 多盘部署
 
-TiFlash 支持多盘部署，主要通过[配置文件 `tiflash.toml`](#配置文件-tiflashtoml) 中的 `path` 和 `path_realtime_mode` 这两个参数控制。
+TiFlash 支持单节点多盘部署。如果你的部署节点上有多块硬盘，可以通过以下的方式配置参数，提高节点的硬盘IO利用率。
+
+#### 全新部署
+
+如果你部署的 TiDB 集群版本 >= v4.0.9：
```

Comment:
```suggestion

- **如果你部署的 TiDB 集群版本 >= v4.0.9：**

```

Reply:
I don't think it is a good idea to define a list following with multiple lines

Reply:
Use subtitle instead of list.